pkg/preparer/orchestrate.go

```diff
@@ -212,6 +206,14 @@ func (p *Preparer) verifySignature(manifest pods.Manifest, logger logging.Logger
 func (p *Preparer) installAndLaunchPod(newManifest *pods.Manifest, pod Pod, logger logging.Logger) bool {
 	// do not remove the logger argument, it's not the same as p.Logger
 
+	// HACK ZONE. When we have better authz, rewrite.
+	if ((newManifest.RunAsUser() == "root") && (newManifest.ID() != POD_ID)) || p.podIdForbidden(newManifest.RunAsUser()) {
+		logger.WithField("pod", newManifest.ID()).
+			Errorln(fmt.Sprintf("Cannot use %s in run_as", newManifest.RunAsUser()))
```

Comment:
A rule of thumb - if you're `sprintf`ing the value into the error message, it should probably be a field in the structured log as well.